hack/deploy-hub-local/ansible/setup.yaml

```diff
@@ -79,3 +79,44 @@
       service:
         name: sshd
         state: restarted
+
+    - name: Add Slack notifications
+      community.general.slack:
+        token: #Add-your-slack-token
+        msg: '{{ inventory_hostname }} completed'
+        channel: #Add-your-slack-channel
+      delegate_to: localhost
+      tags: [ 00-slack ]
+
+    - name: Generate an OpenSSH keypair with the default values (4096 bits, rsa)
```

Comment:
I would leave all the ssh related actions inside the ssh subdirectory, and with the same tag used currently, so we can force/skip it whenever needed.
Also maybe add some vars? For example forcing a certain key type, key length, etc.